source/_integrations/downloader.markdown

```diff
@@ -6,25 +6,16 @@ ha_category:
 ha_release: pre 0.7
 ha_quality_scale: internal
 ha_domain: downloader
+ha_codeowners:
+  - '@erwindouna'
 ha_integration_type: integration
 ---
 
 The `downloader` integration provides a service to download files. It will raise an error and not continue to set itself up when the download directory does not exist. The directory needs to be writable for the user that is running Home Assistant.
```

Comment:
```suggestion
The **Downloader** {% term integration %} provides a service to download files. It will raise an error and not continue to set itself up when the download directory does not exist. The directory needs to be writable for the user that is running Home Assistant.
```